app/Helpers/strings.php

```diff
@@ -25,4 +19,40 @@ function clean_whitespace($input)
         $clear = preg_replace('/ +/', ' ', $clear);
         return $clear;
     }
+
+    /**
+     * Remove any HTML or PHP content for cleanliness
+     * not for security/XSS use md_to_html() instead
+     *
+     * @param string $markdown
+     * @return string untrusted markdown
+     */
+    function prepare_markdown($markdown)
+    {
+        // return strip_tags($markdown);
+        return urldecode(html_entity_decode(clean($markdown, 'strip_all')));
+    }
+
+    /**
+     * Convert markdown to HTML and sanitise HTML through a whitelist (htmlpurifier)
+     *
+     * @param string $markdown
+     * @return string sanitised html
+     */
+    function md_to_html($markdown)
+    {
+        return clean(Markdown::convertToHtml($markdown));
```

Comment:
I'm testing this locally and getting an error if the value is not a string, in my case it's a null because I haven't yet filled out one of the fields in my event. 

`Argument 1 passed to League\CommonMark\Converter::convertToHtml() must be of the type string, null given`

I assume this will be the case anywhere `Markdown::convertToHtml` is called.